### ISSUE TITLE
Transform norm_quote_commas & norm_numbers | Extra inputs are not permitted

### ISSUE THREAD

**Issue by HURIMOZ** [eole-nlp/eole#83]
Hi guys I get an error when using transforms norm_quote_commas and norm_numbers. See below:
```
eole build_vocab --config wmt17_frty.yaml --n_sample -1 # --num_threads 4
Traceback (most recent call last):
  File "/home/ubuntu/TY-EN/TY-EN/bin/eole", line 33, in <module>
    sys.exit(load_entry_point('eole', 'console_scripts', 'eole')())
  File "/home/ubuntu/TY-EN/eole/eole/bin/main.py", line 39, in main
    bin_cls.run(args)
  File "/home/ubuntu/TY-EN/eole/eole/bin/run/build_vocab.py", line 272, in run
    config = cls.build_config(args)
  File "/home/ubuntu/TY-EN/eole/eole/bin/run/__init__.py", line 42, in build_config
    config = cls.config_class(**config_dict)
  File "/home/ubuntu/TY-EN/TY-EN/lib/python3.10/site-packages/pydantic/main.py", line 193, in __init__
    self.__pydantic_validator__.validate_python(data, self_instance=self)
pydantic_core._pydantic_core.ValidationError: 2 validation errors for BuildVocabConfig
data.corpus_1.norm_quote_commas
  Extra inputs are not permitted [type=extra_forbidden, input_value=True, input_type=bool]
    For further information visit https://errors.pydantic.dev/2.8/v/extra_forbidden
data.corpus_1.norm_numbers
  Extra inputs are not permitted [type=extra_forbidden, input_value=True, input_type=bool]
    For further information visit https://errors.pydantic.dev/2.8/v/extra_forbidden
```

**Comment by francoishernandez** [comment]
The config you provided is apparently not the one which triggered the error, but investigating further lead me to the issue.
Should be fixed in #87. Can you try this branch to make sure it fixes all your issues before we merge?

**Comment by HURIMOZ** [comment]
Hi François, yes the error is gone now. Thanks for that.
